yarn.lock

```diff
@@ -2929,6 +2929,11 @@
   dependencies:
     semver "^7.3.7"
 
+"@safe-global/safe-gateway-typescript-sdk@^3.19.0":
+  version "3.22.2"
+  resolved "https://registry.yarnpkg.com/@safe-global/safe-gateway-typescript-sdk/-/safe-gateway-typescript-sdk-3.22.2.tgz#d4ff9972e58f9344fc95f8d41b2ec6517baa8e79"
+  integrity sha512-Y0yAxRaB98LFp2Dm+ACZqBSdAmI3FlpH/LjxOZ94g/ouuDJecSq0iR26XZ5QDuEL8Rf+L4jBJaoDC08CD0KkJw==
```

Comment:
Strange that this appeared. Is it due to `yarn install`?

Reply:
yes